datenguide_python/query_builder.py

```diff
@@ -76,34 +130,32 @@ def _get_fields_to_query(self, field: Union[str, "Field"]) -> str:
 
             if field.fields:
                 substring += "{"
-                for subfield in field.fields:
-                    substring += field._get_fields_to_query(subfield)
+                for key, field_item in field.fields.items():
+                    substring += field._get_fields_to_query(field_item)
                 substring += "}"
         else:
             raise TypeError
         return substring
 
     def get_fields(self):
         field_list = [self.name]
-        for field in self.fields:
-            field_list.extend(Field._get_fields_helper(field))
+        for key, value in self.fields.items():
```

Comment:
`key` not used as well
```suggestion
        for value in self.fields.values():
```